test/web-platform-tests/to-run.yaml

```diff
@@ -824,3 +824,15 @@ failure.html: [fail, Fetch is not implemented]
 toascii.window.html: [fail, Unknown]
 url-constructor.html: [fail, url.search vs. url.searchParams propagation]
 urlencoded-parser.html: [fail, Fetch is not implemented]
+
+---
+
+DIR: websockets
+
+Create-Secure-extensions-empty.htm: [fail, Buggy test as the test does not take into account the mandatory permessage-deflate extension]
+Secure-Close-1005-verify-code.htm: [fail, See https://github.com/websockets/ws/issues/1257]
+closing-handshake/003.html*: [fail, See https://github.com/websockets/ws/issues/1257]
+interfaces/WebSocket/close/close-basic.html*: [fail, Mutates globals]
```

Comment:
This test passes because we implement things correctly, but because the wrappers are shared across JSDOM windows it will affect the next test, thus causing failures. (This one was a pain to debug as one can imagine.)

Reply:
I know at least one more test which behaves this way, though it's among the CSSOM tests and not included here yet. Annoying to debug, indeed.

Since we eventually want to run most failing tests to make sure that they're still failing, we ought to add a new failure reason (`[mutates-globals]` or something more general?) for these tests to make sure they won't run and affect others later. That doesn't need to be done in this pull request though.

Reply:
Agreed on all counts. 

We also need a new failure reason for flaky tests. I experienced one of them while auditing which tests we could enable after the WPT roll.